paymentrails.gemspec

```diff
@@ -12,5 +12,7 @@ Gem::Specification.new do |s|
   s.files = Dir.glob ["README.rdoc", "LICENSE", "lib/**/*.{rb,crt}", "spec/**/*", "*.gemspec"]
   s.required_ruby_version = '>= 2.4'
   s.add_dependency "rest-client", ">= 2.0.0"
+  s.add_development_dependency 'rake', '~> 12'
   s.add_development_dependency "rubocop", '~> 0.77'
+  s.add_development_dependency 'test-unit', '~> 3'
```

Comment:
Ruby has gone through the process of gem-ifying things that were formerly part of core, so that's why this is here